core/src/main/java/tech/pegasys/web3signer/core/multikey/SignerLoader.java

```diff
@@ -54,6 +55,8 @@
           .collect(Collectors.toSet());
     } catch (final IOException e) {
       LOG.error("Unable to access the supplied key directory", e);
+    } finally {
+      InterlockKeyProvider.INSTANCE.closeAllSessions();
```

Comment:
do not love the singleton, can we pass an InterlockKeyProvider into the AbstractArtifactSignerFactory at startup (instead of vertx)? Then there's only one of them.